python/pyspark/sql/readwriter.py

```diff
@@ -692,8 +734,7 @@ def orc(self, path, mode=None, partitionBy=None, compression=None):
                             This will override ``orc.compress``. If None is set, it uses the
                             default value, ``snappy``.
 
-        >>> orc_df = spark.read.orc('python/test_support/sql/orc_partitioned')
```

Comment:
Ah sorry, I was going to look into making the test do the lucene random testing thing of switching between the dataformats provided for `df` randomly.

I was going to change the runner to use `random.choice` to pick between orc and parquet (and you know one day arrow, hdf5 whatever).